assignment.html

```diff
@@ -0,0 +1,84 @@
+<html>
+    <head>
+        <style>
+            figure
+            {
+                float:left;
+                border-left: 1px solid rgba(228, 223, 223, 0.87);
+                height: 300px;
+                margin-left:-50px;
+                margin-right:59px;
+                margin-top:-20px; 
+            }
+            img
+            {
+                width:200px;
+                height:200px;
+                margin:10px;
+            }
+            figcaption
+            {
+                width:200px;
+                margin-left: 9px;
+            }
+            .d
+            {
+                color:dimgray;
+                font-size: 12px;
+            }
+            a
+            {
+                text-decoration: none;
+                color:black;
+                text-decoration-style: solid;
```

Comment:
text-decoration and text-decoration-style can be combined into one 

`
text-decoration: text-decoration-line text-decoration-color text-decoration-style|initial|inherit;
`